docs/src/components/DocLayout/index.tsx

```diff
@@ -113,7 +113,9 @@ export default function DocLayout({
               display: flex;
               margin: 0 auto;
               width: 100%;
-              max-width: 80rem;
+              box-sizing: border-box;
+              padding: 0 2rem;
+              max-width: 90rem;
```

Comment:
We can use the `MAX_CONTENT_WIDTH` global const, no?

Reply:
yes